renderer/components/TimetrackerWebsiteConncetion.tsx

```diff
@@ -154,6 +154,7 @@ const TimetrackerWebsiteConnection = () => {
 
       const timtrackerYearProjects = userFetchedData[2];
       setProjects(timtrackerYearProjects);
+      localStorage.setItem("year-projects", timtrackerYearProjects.join(","));
       userInfo.yearProjects = timtrackerYearProjects;
 
       localStorage.setItem("timetracker-user", JSON.stringify(userInfo));
```

Comment:
you don't need to set in lcoalStorage "year-projects" bacause ive already added it in userInfo obj in 158 line, later in 160 i set it into localStorage.